packages/owl-bot/cron.yaml

```diff
@@ -14,4 +14,7 @@ cron:
     description: owl-bot syncing labels
     params:
       cron_type: "repository"
+      allowed_organizations:
+        - "googleapis"
+        - "googlecloudplatform"
```

Comment:
hopefully, this gets converted correctly in the requests

Reply:
will try